src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -918,6 +919,28 @@ private LoadFlowParametersEntity createParametersEntity(LoadFlowParametersValues
         return LoadflowService.toEntity(allCommonValues, allSpecificValues);
     }
 
+    public SecurityAnalysisParametersValues getSecurityAnalysisParametersValues(UUID studyUuid) {
+        return studyRepository.findById(studyUuid)
+                .map(studyEntity -> studyEntity.getSecurityAnalysisParameters() != null ? formattedSecurityAnalysisParametersValues(SecurityAnalysisService.fromEntity(studyEntity.getSecurityAnalysisParameters()), true) : formattedSecurityAnalysisParametersValues(SecurityAnalysisService.getDefaultSecurityAnalysisParametersValues(), true))
+                .orElse(null);
+    }
+
+    public SecurityAnalysisParametersValues formattedSecurityAnalysisParametersValues(SecurityAnalysisParametersValues values, boolean isFromEntity) {
```

Comment:
- naming: should be formatSecurityAnalysisParametersValues or securityAnalysisParametersFormattedValues
- design: to my opinion, the back must not format the values, it's up to the Front

Reply:
Indeed. From  now on formatting is done in front-end.